fabtests/pytest/common.py

```diff
@@ -1,9 +1,15 @@
 import pytest
 import errno
 
-def has_cuda(ip):
+def has_cuda_hmem_support(cmdline_args, host_type, ip):
```

Comment:
Can we split this function into two, `has_cuda` and `has_hmem_support`? In the future we will implement similar things for other hmem ifaces and they can reuse `has_hmem_support` .

Reply:
Sure